app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -281,7 +283,7 @@ def inspector_admin_secret
       # TODO: support multiple imagePullSecrets. This depends on image-inspector support
       return inspector_sa.try(:imagePullSecrets).to_a[0].try(:name)
     rescue KubeException => e
-      raise e unless e.error_code == 404
+      raise e unless e.error_code == ERRCODE_NOTFOUND
```

Comment:
But on socket error i do want to raise the exception??
I want to ignore it only if the secret does not exist.